src/main/java/org/breedinginsight/daos/ExperimentalCollaboratorDAO.java

```diff
@@ -36,4 +42,22 @@ public ExperimentalCollaboratorDAO(Configuration config, DSLContext dsl) {
         super(config);
         this.dsl = dsl;
     }
+
+    public ExperimentProgramUserRoleEntity create(UUID experimentId, UUID programUserRoleId, UUID userId) {
```

Comment:
Was the primary reason for doing it this way rather than using the DAO `insert` to have the data for returning without needing to do a select query after writing?

Reply:
Exactly.